src/bindings/python/flux/job/directives.py

```diff
@@ -49,6 +49,11 @@ def __init__(self, value, lineno=-1):
         # quoting and avoid splitting on cmdline args like --foo.
```

Comment:
cut and paste typo "negatel"

Reply:
Thanks! Fixed that and pushed the result.